.gitignore

```diff
@@ -7,6 +7,8 @@ __pycache__/
 /*.egg
 /*.egg-info/
 /*.egg-link
+.eggs/
+.pytest_cache/
```

Comment:
This should be rooted in the project directory, I assume, thus start with a slash.

Please put this into a commit of its own.